apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx

```diff
@@ -111,26 +111,28 @@ export default class Results extends React.Component {
   }
 
   renderFacilitatorAverages() {
-    return Object.keys(this.props.facilitators).map((facilitator_id, i) => (
-      <Tab
-        eventKey={this.props.sessions.length + i + 1}
-        key={i}
-        title={this.props.facilitators[facilitator_id]}
-      >
-        <FacilitatorAveragesTable
-          facilitatorAverages={
-            this.props.facilitatorAverages[
-              this.props.facilitators[facilitator_id]
-            ]
-          }
-          facilitatorId={parseInt(facilitator_id, 10)}
-          facilitatorName={this.props.facilitators[facilitator_id]}
-          questions={this.props.facilitatorAverages['questions']}
-          courseName={this.props.courseName}
-          facilitatorResponseCounts={this.props.facilitatorResponseCounts}
-        />
-      </Tab>
-    ));
+    return Object.keys(this.props.facilitators || {}).map(
```

Comment:
This line is the only content change, the rest of this file is just whitespace change.

Reply:
Under what circumstances is the `facilitators` prop falsy?  It's still required according to the proptypes:
```jsx
facilitators: PropTypes.object.isRequired,
```
If you do want to safely omit this prop, perhaps a default value for the prop (e.g. `static defaultProps = ...`) is a better solution than this inline default.

Reply:
`facilitators` field is created only when doing survey roll-up. Not all workshops require doing survey roll-up; an example is CSF deep dive. Previously, `DailySurveyDecorator` always return an empty hash for this field just to please the UI. I remove that unnecessary code, resulting in undefined `facilitators` value on the client side. 

Will use `defaultProps` for fields that only come from survey roll-up.